### PR TITLE
i18n(fr): Adding `deploy/sst` French translation

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)

Adding `deploy/sst` French translation. That it!

I am participating in the Hacktoberfest! 🚀☕ (*Yes, I alternate emojis because it's always the same message.*)
- [i18n(fr): Adding deploy/sst French translation](https://github.com/withastro/docs/commit/d40b9a6347d2e0d146eb3ca6e65b3b946ee5ea61)

### CODE REVIEW

File: src/content/docs/fr/guides/deploy/sst.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+Vous pouvez déployer un site Astro en utilisant [SST](https://sst.dev), un framework open-source pour déployer des applications entièrement serverless sur AWS avec prise en charge des SSG et SSR.
+
+Vous pouvez également utiliser toute construction SST supplémentaire comme les Cron Jobs, les Buckets, les Queues, etc. tout en maintenant la sécurité de type.
+
+## Démarrage rapide
+
+1. Créer un projet astro
+2. Exécutez `npx create-sst`
+3. Il devrait détecter que vous utilisez Astro et vous demander de confirmer.
+4. Une fois que vous êtes prêt pour le déploiement, vous pouvez lancer `npx sst deploy --stage=production`
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/deploy/sst.mdx:17]:
```suggestion
1. Exécutez `npx create-sst`
1. Il devrait détecter que vous utilisez Astro et vous demander de confirmer.
1. Une fois que vous êtes prêt pour le déploiement, vous pouvez lancer `npx sst deploy --stage=production`
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Bon travail @thomasbnt. Voici un mineur suggestion pour être consistant avec le docs en anglais.

**Comment by McFlyPartages** [comment]
LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work everyone 🚀

**Pull request merged**